crates/holochain/src/core/queue_consumer/countersigning_consumer.rs

```diff
@@ -1,43 +1,48 @@
 //! The queue consumer for the countersigning workflow.
 
 use super::*;
+#[cfg(feature = "unstable-countersigning")]
 use crate::core::workflow::countersigning_workflow::countersigning_workflow;
 use tracing::*;
 
 /// Spawn the QueueConsumer for the countersigning workflow
 #[instrument(skip_all)]
 pub(crate) fn spawn_countersigning_consumer(
-    space: Space,
-    workspace: Arc<CountersigningWorkspace>,
+    _space: Space,
+    _workspace: Arc<CountersigningWorkspace>,
     cell_id: CellId,
     conductor: ConductorHandle,
-    integration_trigger: TriggerSender,
-    publish_trigger: TriggerSender,
+    _integration_trigger: TriggerSender,
+    _publish_trigger: TriggerSender,
 ) -> TriggerSender {
     let (tx, rx) = TriggerSender::new();
 
-    let self_trigger = tx.clone();
+    let _self_trigger = tx.clone();
     queue_consumer_cell_bound(
         "countersigning_consumer",
         cell_id.clone(),
         conductor.task_manager(),
         (tx.clone(), rx),
+        #[cfg(not(feature = "unstable-countersigning"))]
```

Comment:
This is the main prevention for the workflow to ever be run. The queue consumer never registers the actual cs workflow function.